lib/Common/CommonDefines.h

```diff
@@ -131,33 +146,27 @@
 #define ENABLE_BACKGROUND_PAGE_ZEROING 1
 #define ENABLE_BACKGROUND_PAGE_FREEING 1
 #define ENABLE_RECYCLER_TYPE_TRACKING 1
+#define SUPPORT_SLIST 1
```

Comment:
here SLIST means the nonblocking implementation of interlocked singly linked list  in win32 , let me change it to SUPPORT_WIN32_SLIST

---
In reply to: [87880167](https://github.com/Microsoft/ChakraCore/pull/1968#discussion_r87880167) [](ancestors = 87880167)